deploy/packages/bbp-packages.yaml

```
@@ -37,6 +37,7 @@ packages:
       - py-morphology-repair-workflow@1.0.4
       - py-simwriter@1.0.1
       - py-morph-tool@0.1.14
+      - py-morphio
```

```suggestion
      - py-morphio@2.3.4
```
A.k.a. the latest version. Otherwise, our deployment will *always* generate modules for all installed versions, not just the latest one.